container/public/LuigiCompoundContainer.svelte.d.ts

```diff
@@ -1,3 +1,13 @@
 export default class LuigiCompoundContainer extends HTMLElement {
   compoundConfig: any;
+
+  /**
+   * If set to `true`, a micro frontend will be immediately initialized. LuigiContainer sends an event `initialized` to the micro frontend.
+   */
+  instantInit: boolean;
```

Comment:
should we maybe make this **instantinit** also, same as in `LuigiContainer.svelte`